std/experimental/logger/core.d

```diff
@@ -12,56 +12,6 @@ import std.traits;
 
 import std.experimental.logger.filelogger;
 
-/** This template evaluates if the passed `LogLevel` is active.
-The previously described version statements are used to decide if the
-`LogLevel` is active. The version statements only influence the compile
-unit they are used with, therefore this function can only disable logging this
-specific compile unit.
-*/
-template isLoggingActiveAt(LogLevel ll)
-{
-    version (StdLoggerDisableLogging)
-    {
-        enum isLoggingActiveAt = false;
```

Comment:
shouldn't we issue a deprecation when this version is used?

Reply:
It's an experimental module.

Reply:
what Iain said